src/controllers/handlers/wert-signer-handler.ts

```diff
@@ -0,0 +1,33 @@
+import { WertMessage } from '../../ports/wert-signer/types'
+import { HTTPResponse, HandlerContextWithPath, StatusCode } from '../../types'
+
+export async function createWertSignerHandler(
+  context: Pick<HandlerContextWithPath<'wertSigner', '/v1/wert/sign'>, 'url' | 'components' | 'params' | 'request' | 'verification'>
+): Promise<HTTPResponse<string>> {
+  const {
+    verification,
+    request,
+    components: { wertSigner }
+  } = context
+  const userAddress: string | undefined = verification?.auth.toLowerCase()
+  const body: WertMessage = await request.json()
+
+  if (!userAddress || (!!userAddress && userAddress !== body.address)) {
```

Comment:
is the `body.address` always lowercase?

Reply:
mmm the tests I've done yes, since it comes from `wallet.address` in the UI. But it can happen that comes out from other place and it's not. Updated!